frontend/prebuild/src/app/game/game.service.ts

```diff
@@ -18,20 +18,20 @@ export class GameService {
 
     socketService.url = `${environment.API_URL_GAME_WS}/${this.roundId}`;
     this.messages = <Subject<Object>>socketService.socket
-    .map((response: MessageEvent): any => {
-      //console.log(`Game service handling message: ${response.data}`);
+    .pipe(map((response: MessageEvent): any => {
+      console.log(`Game service handling message: ${response.data}`);
```

Comment:
can we leave this log statement commented out?  It fills up logs very fast

Reply:
Don’t know how it got put back in. Must have been a bad merge. I can fix it.